tutor/test_cc1_14_TrainingAndSupportingTeachersAndStudents.py

```diff
@@ -86,7 +91,21 @@ def test_system_concept_coach_zendesk_is_web_accessible_7704(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
+        self.teacher.login()
+        self.teacher.driver.find_element(
+            By.XPATH, '//a[contains(@href,"/cc-dashboard/")]'
+        ).click()
+        self.teacher.open_user_menu()
+        self.teacher.driver.find_element(
+            By.LINK_TEXT, 'Get Help'
+        ).click()
```

Comment:
Like CC1-01, use the internal functions to the user or assignment:
`self.<user>.find(<by>, <string>)`